src/frontend/data_structures/lists/index.js

```diff
@@ -1,5 +1,5 @@
 define(["d3js", "variable"], function(){
-  function drawSector(list, available) {}
+  function drawSector(svg, list, available) {}
```

Comment:
Very nice!
1. The border covering variable values is less thick on the upper side than everywhere else.
2. I'd rather fill affected variables with color instead of changing the border.